packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -130,6 +131,15 @@ export default class Server {
         };
     }
 
+    async onInitialized(): Promise<void> {
+        // The config value comes from salesforcedx-vscode/salesforcedx-vscode-lwc
+        const hasTsEnabled = await this.connection.workspace.getConfiguration('salesforcedx-vscode-lwc.preview.typeScriptSupport');
```

Comment:
The entire feature is gated behind this feature flag, which hasn't been implemented in salesforcedx-vscode yet.

There is a follow-up PR to implement the flag, in the meantime, I've tested locally that when the config value is enabled it works correctly.

Reply:
After discussing with the team, we should remove the `tsconfig` if `salesforcedx-vscode-lwc.preview.typeScriptSupport = false`.